packages/atomic/src/components/commerce/atomic-commerce-interface/atomic-commerce-interface.tsx

```diff
@@ -166,7 +166,9 @@ export class AtomicCommerceInterface
   }
```

Comment:
Out of scope, but these exceptions when the atomic-commerce-interface's type is "recommendations" make me think we may want to have a dedicated simplified interface component for this use case after all.